docs/source/examples.rst

```diff
@@ -270,3 +270,39 @@ You can also pass in a file-like object with:
 
 To extract metadata from ``.docx`` or ``.xlsx``, use ``get_docx_metadata`` and
 ``get_xlsx_metadata``. The interfaces are the same as ``get_jpg_metadata``.
+
+
+###########################
+Exploring Source Documents
+###########################
+
+The ``unstructured`` library includes tools for helping you explore source documents.
+To get a summary of the size and type of documents in a directory, you can
```

Comment:
What's meant by "size" here? I don't see file sizes in the summary...

Reply:
If you run `file_info.head()` you'll see it. That's the size of the file in bytes. I'll update the docs to clarify